cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -85,19 +85,19 @@ object ScannerCommand : CommandWithHelp() {
             order = PARAMETER_ORDER_OPTIONAL)
     private var scannerFactory = ScanCode.Factory()
 
-    @Parameter(description = "The path to the configuration file.",
+    @Parameter(description = "The path to a configuration file.",
             names = ["--config", "-c"],
             order = PARAMETER_ORDER_OPTIONAL)
     private var configFile: File? = null
 
-    @Parameter(description = "The list of output formats used for the result file(s).",
+    @Parameter(description = "The list of output formats to be used for the ORT result file(s).",
             names = ["--output-formats", "-f"],
             order = PARAMETER_ORDER_OPTIONAL)
     private var outputFormats = listOf(OutputFormat.YAML)
 
     override fun runCommand(jc: JCommander) {
         require((dependenciesFile == null) != (inputPath == null)) {
-            "Either --analyzer-result-file or --input-path must be specified."
+            "Either '--ort-file' or '--input-path' must be specified."
```

Comment:
out of scope: would it make sense to refactor this - e.g. allow just a single way of specifying the ort file?